packages/eslint-config-jest/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * strvcom/eslint-config-jest
+ *
+ * @author      Michal Jarnot <m.jarnot@yahoo.com>
+ * @copyright   2019 STRV
+ * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
+ */
+
+'use strict'
+
+module.exports = {
+  extends: ['plugin:jest-formatting/recommended'],
+  plugins: ['jest', 'jest-formatting'],
+  overrides: [{
+    files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
+    env: {
+      'jest/globals': true,
+    },
+  }],
+  rules: {
+    //
+    // Recommended
+    //
+
+    'jest/expect-expect': 'error',
+    'jest/no-disabled-tests': 'error',
```

Comment:
not exclusive to this one but I kinda feel like the whole config is full of `error` where in many cases I would actually prefer to have `warn`. I came to believe that in the real world usage having this to be so strict is ignoring a lot of valid cases that may warrant a warning :warning: , but are not really something I would strictly eliminate as erroneous practice

Reply:
This is the same issue as with comments. There is no reason to commit tests that don't do anything (skipped, commented out)

What I do when I refactor code or have to rewrite feature:
- fix test immediately
- remove broken test and rewrite the test later (it is in git history, there is not a reason to have tests in a codebase that don't do anything)

The worst-case scenario is when people use `skip` because they want to test something in the future. That is likely not going to happen. 🙂 

With that said this is supposed to be STRV config and it doesn't have to be that strict if majority of people prefer linter with warnings instead of errors.

Reply:
@hudecsamuel If you don't like some rules, please leave a comment with the reason why it should (not) be included. This config ruleset is for everyone. 🙂

Reply:
> remove broken test and rewrite the test later (it is in git history, there is no reason to have code in a codebase that doesn't do anything)

This is most certainly not helpful at all in practice. When you have a "pending"/skipped test clearly visible in the CI logs it's a very good and strong reminder to go and look at it eventually. If you delete the test completely you are most likely going to completely forget about it. ⚠️